packages/Button/stories/Button.mdx

```diff
@@ -0,0 +1,234 @@
+import PlusIcon from "@paprika/icon/lib/Add";
+import InfoIcon from "@paprika/icon/lib/InfoCircle";
+import ButtonSubmit from "./examples/ButtonSubmit";
+import CloseButtonRef from "./examples/CloseButtonRef";
+import Focus from "./examples/Focus";
+import NewRef from "./examples/NewRef";
+import OldRef from "./examples/OldRef";
+import Button from "../src/";
+
+## More Examples
+
+### Index
+
+- <a href="#basic-button">Basic Button</a>
+
+  - <a href="#kinds">Kinds</a>
+
+  - <a href="#sizes">Sizes</a>
+
+  - <a href="#states">States</a>
+
+  - <a href="#submit">Submit</a>
+
+- <a href="#close-button">Close Button</a>
+- <a href="#link-button">Link Button</a>
+- <a href="#icon-button">Icon Button</a>
+
+### Basic Button
+
+To add a basic button import the button and add the `<Button>` component to your project.
+
+```js
+<Button>Click Me!</Button>
+```
+
+##### Result
+
+<Button>Click Me!</Button>
+
+#### Kinds
+
+Differant projects require differant buttons, luckily the Button component can make us of the `kind` property to create a variety of buttons for differant use cases.
+
+```js
+<Button kind="Button.types.kind.DEFAULT">default</Button>
+<Button kind="Button.types.kind.PRIMARY">primary</Button>
+<Button kind="Button.types.kind.SECONDARY">secondary</Button>
+<Button kind="Button.types.kind.DESTRUCTIVE">destructive</Button>
+<Button kind="Button.types.kind.FLAT">flat</Button>
+<Button kind="Button.types.kind.MINOR">minor</Button>
+<Button kind="Button.types.kind.LINK">link</Button>
+```
+
+##### Result
+
+<Button kind="default">default</Button>
+<Button kind="primary">primary</Button>
+<Button kind="secondary">secondary</Button>
+<Button kind="destructive">destructive</Button>
+<Button kind="flat">flat</Button>
+<Button kind="minor">minor</Button>
+<Button kind="link">link</Button>
+
+#### Sizes
+
+Buttons can be adjusteds to be a variety of sizes using the `size` proeprty or the `isFullWidth` proeprty.
+
+```js
+<Button size="Button.types.size.SMALL">small</Button>
+<Button size="Button.types.size.MEDIUM">medium</Button>
+<Button size="Button.types.size.LARGE">large</Button>
+<Button isFullWidth>full width</Button>
+```
+
+##### Result
+
+<Button size="small">small</Button>
+<Button size="medium">medium</Button>
+<Button size="large">large</Button>
+<Button isFullWidth>full width</Button>
+
+#### States
+
+Buttons can be display differant states by adjusting making use of the `isActive`, `isPending`, and `isDisabled` properties.
+
+```js
+    <Button isActive>active</Button>
+    <Button isDisabled>disabled</Button>
+    <Button isPending>pending</Button>
+```
+
+##### Result
+
+<Button isActive>active</Button>
+<Button isDisabled>disabled</Button>
+<Button isPending>pending</Button>
+
+#### Submit
+
+To turn a Button into a submit Button by adding the `isSubmit` property.
+
+```js
+<form
+  onSubmit={event => {
+    alert("submitted");
+    event.preventDefault();
+  }}
+>
+  <input type="text" defaultValue="press enter while focus" />
```

Comment:
It's so ugly 🙈  

The Paprika `<Input>` would be much nicer, but you'd have to change the `<ButtonSubmit>` story too, and add some custom CSS... 😓